backend/controller/ingress/request_test.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 type AliasRequest struct {
-	Aliased string `json:"alias"`
+	Aliased string
```

Comment:
@alecthomas I removed `json:"alias"` here mostly to demonstrate what the error was for the query params test. I believe this is related to #951 . Because in this test, we use `json.Unmarshal` which will use the `json` alias here. Since `BuildRequestBody` will have already handled the aliasing and will now have the field name in the map as `Aliased` when it `json.Unmarshal`s it will lose that field and we end up with `Aliased: ` instead of `Aliased: value`.

I think we'll want this test to use `encoding.Unmarshal` instead of `json.Unmarshal` once it's available. 

In the meantime, we can use this code, which fixes the tests or put it backed the way it was (commented out).